src/app/contextBroker/contextBroker.cpp

```diff
@@ -719,7 +721,9 @@ static int loadFile(char* path, char* out, int outSize)
     return -1;
```

Comment:
[My fault] This line needs to return NULL and not -1.

There's one more "return -1" in the function that also (of course) needs the same change